### ISSUE TITLE
Add new Feature Request - Support Stellite's algorithm change

### ISSUE THREAD

**Issue by adi3yz** [Bendr0id/xmrigCC#105]
i fan XTL now.. pls add this...
Feature Request - Support Stellite's algorithm change from CryptonightV1 to a slightly modified variant
https://github.com/fireice-uk/xmr-stak/issues/1494
https://github.com/xmrig/xmrig/issues/581

**Comment by djfinch** [comment]
Guys... No need to change anything. Stellite working out of the box. Just use
`"force-pow-version": 2,`. I'm using it for weeks...

**Comment by Bendr0id** [comment]
@djfinch, you are right. For now this is the right solution. But stellite will fork again. Then modified it slightly. I'll implement it soon.

Am 24.04.2018 18:54 schrieb djfinch <notifications@github.com>:

Guys... No need to change anything. Stellite working out of the box. Just use
"force-pow-version": 2,. I'm using it for weeks...

**Comment by djfinch** [comment]
Damn, I absolutely missed this. Seems it's a beginning of algo/variant hell...
So for now we can be safe. Not sure which height will trigger this fork... Have to investigate...

**Comment by stellitedev** [comment]
@Bendr0id is right, we already implemented v7 at height 100800, we will be forking again soon to a slightly modified variant. It currently is just on a testnet [here](https://testxtl.anypool.net/#), but once we get enough miners support we will set a fork height and announce the fork.

**Comment by Bendr0id** [comment]
I already started with the integration of the new PoW. In currently on vacation, but I hope to get it done within the next days. Do you have some test vectors included in the source?



@Bendr0id<https://github.com/Bendr0id> is right, we already implemented v7 at height 100800, we will be forking again soon to a slightly modified variant. It currently is just on a testnet here<https://testxtl.anypool.net/#>, but once we get enough miners support we will set a fork height and announce the fork.

**Comment by bobbieltd** [comment]
@Bendr0id Can you add support for variant like xmrig ? For example : algo : “cryptonight-like”, variant : “ipbc”; algo : “cryptonight” , variant : “xtl”.
I have not tested to use "force-pow-version" as “variant” for “ipbc”. I guess you don’t support it yet.
